homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -520,7 +527,9 @@ async def put(self, request, username, entity_number):
             # they'll map to "on". Thus, instead of reporting its actual
             # status, we report what Alexa will want to see, which is the same
             # as the actual requested command.
-            config.cached_states[entity_id] = parsed
+            config.cached_states[entity_id] = [parsed, None]
+        else:
+            config.cached_states[entity_id] = [parsed, time.time()]
```

Comment:
We should probably move this code (just the part about setting the cache) after the `wait_for_state_change_or_timeout` so the cache lasts for the full 2 seconds after the successful state change.

Reply:
That should better handle your group case.

Reply:
I did some testing with groups, and it is not longer enough.  I made the adjustment.